docs/cpp_standards.md

```diff
@@ -27,43 +23,3 @@ To get C++17 support, Xcode 9.3+ is needed, requiring at least macOS 10.13 on th
 However, if only C++17 compiler and standard template library (STL) features are used (not needing a C++17 runtime) it might be possible to set `MACOSX_DEPLOYMENT_TARGET` to a lower value, such as `"10.9"`. To find out if this is the case, try compiling and running with a lower `MACOSX_DEPLOYMENT_TARGET`: if C++17 features are used that require a more recent deployment target, building the wheel should fail.
 
 For more details see https://en.cppreference.com/w/cpp/compiler_support, https://en.wikipedia.org/wiki/Xcode, and https://xcodereleases.com/: Xcode 10 needs macOS 10.13 and Xcode 11 needs macOS 10.14.
-
-## Windows and Python 2.7
```

Comment:
This is a bit sad that we have to let go of all this information, though? It would be fine if it were indicated clearly somewhere else, but we struggled to put bits and pieces together and figure out how to do this.

But yes, if `cibuildwheel` doesn't support 2.7 anymore, there's no point keeping these kind of workarounds. It's just said to lose this information?

Reply:
Mayte still keep short text with link that building python 2.7 wheels was supported in version 1.9 and link to its documentation?

Reply:
Maybe, yes. But then still 1.9 might not be around forever? :-/

Reply:
I think we should have a 1.9.x branch and add that to our docs versions, so users can switch to the 1.9.x docs.

Reply:
1.9 LTS will by definition be around until 2.7 is, right? That's the downside of (responsibly) dropping something but keeping an LTS, you now maintain two branches.